### DISCUSSION TITLE
How to exclude some directories from files search?

### DISCUSSION THREAD

**Discussion by Em-AK** [liuchengxu/vim-clap#620] [Q&A] [answered]
When I am in a git repo with lots[ __*__ ] of vendor dependencies copied in the repo, then files search results are burried in a lot of noise and search input becomes unresponsive.

How could I exclude these directories from files search results?
Is there an option I could activate per repo in my .vimrc?

[ __*__ ] like 100 times more LOC than the rest of the repo.

**Comment by liuchengxu** [comment, marked as answer]
`:h :clap-files`, pass the options of `fd/rg/git/find` accordingly.

**Reply by Em-AK** [reply]
Thanks for the hint.
I ended up adding an `.ignore` file in the repo to complement / revert the content of `.gitignore`

https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md#automatic-filtering
